src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Builds.cs

```diff
@@ -16,8 +16,8 @@ public class ScenarioTests_Builds : MaestroScenarioTestBase
         private readonly string repoName = TestRepository.TestRepo1Name;
         private readonly string sourceBuildNumber = "654321";
         private readonly string sourceCommit = "123456";
-        private readonly string sourceBranch = "master";
-        
+        private readonly string sourceBranch = "main";
```

Comment:
These tests target repos in this org: https://github.com/maestro-auth-test and the repos there don't have a main branch. 

I don't think it's worth changing those repos to main, but there's nothing stopping us from doing it. The change would need to happen in all the repos before this gets merged.

Reply:
To be doubly sure, we should run the scenario tests locally with this change. https://github.com/dotnet/arcade-services/blob/master/docs/DevGuide.md#running-scenario-tests-against-a-local-cluster